internal/controller/pipelinerollout/pipelinerollout_ppnd.go

```diff
@@ -64,23 +64,15 @@ func (r *PipelineRolloutReconciler) processExistingPipelineWithPPND(ctx context.
 		return false, errors.New("not enough information available to know if we need to pause")
 	}
 
-	resumed := false
 	shouldBePaused := *needsPaused
 	if shouldBePaused {
 		if err := r.setPipelineLifecyclePaused(ctx, existingPipelineDef); err != nil {
 			return false, err
 		}
 	} else {
-		// make sure this is set to running
 		if err := r.setPipelineLifecycleRunning(ctx, pipelineRollout, existingPipelineDef, forceResume); err != nil {
 			return false, err
 		}
-
```

Comment:
this was code I had to put in in [this](https://github.com/numaproj/numaplane/pull/833/files#diff-00bcf30e52375230575763d7076594d12990dd5c3e8b5654d528d558ac664d35) PR, but it was only necessary because we were making a change to the pipeline "resume-strategy" then as the way to control the Pipeline and we needed to make sure to reset it afterward. We no longer need to do any of that.